specification/elastic/resource-manager/Microsoft.Elastic/stable/2023-06-01/elastic.json

```diff
@@ -0,0 +1,2510 @@
+        },
+        "nextLink": {
+          "description": "Link to the next set of results, if any.",
+          "type": "string"
+        }
+      }
+    },
+    "MarketplaceSaaSInfo": {
+      "description": "Marketplace SAAS Info of the resource.",
+      "type": "object",
+      "properties": {
+        "marketplaceSubscription": {
+          "description": "Marketplace Subscription",
+          "type": "object",
+          "properties": {
+            "id": {
+              "description": "Marketplace Subscription Id. This is a GUID-formatted string.",
+              "type": "string"
+            }
+          }
+        },
+        "marketplaceName": {
+          "description": "Marketplace Subscription Details: SAAS Name",
+          "type": "string"
+        },
+        "marketplaceResourceId": {
+          "description": "Marketplace Subscription Details: Resource URI",
+          "type": "string"
+        },
+        "marketplaceStatus": {
+          "description": "Marketplace Subscription Details: SaaS Subscription Status",
+          "type": "string"
+        }
+      }
+    },
+    "ElasticDeploymentStatus": {
+      "description": "Flag specifying if the Elastic deployment status is healthy or not.",
+      "enum": [
+        "Healthy",
+        "Unhealthy"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "ElasticDeploymentStatus"
+      }
+    },
+    "DeploymentInfoResponse": {
+      "description": "The properties of deployment in Elastic cloud corresponding to the Elastic monitor resource.",
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "The Elastic deployment status.",
+          "$ref": "#/definitions/ElasticDeploymentStatus",
+          "readOnly": true
+        },
+        "version": {
+          "description": "Version of the elasticsearch in Elastic cloud deployment.",
+          "type": "string",
+          "readOnly": true
+        },
+        "memoryCapacity": {
+          "description": "RAM capacity of the elasticsearch in Elastic cloud deployment.",
+          "type": "string",
+          "readOnly": true
+        },
+        "diskCapacity": {
+          "description": "Disk capacity of the elasticsearch in Elastic cloud deployment.",
+          "type": "string",
+          "readOnly": true
+        },
+        "deploymentUrl": {
+          "description": "Deployment URL of the elasticsearch in Elastic cloud deployment.",
+          "type": "string",
+          "readOnly": true
+        },
+        "marketplaceSaasInfo": {
+          "description": "Marketplace SaaS Info of the resource.",
+          "$ref": "#/definitions/MarketplaceSaaSInfo",
+          "readOnly": true
+        }
+      }
+    },
+    "ExternalUserCreationResponse": {
+      "description": "The properties of the response we got from elastic while creating external user",
+      "type": "object",
+      "properties": {
+        "created": {
```

Comment:
>created [](http://example.com/codeflow?start=9&length=7)

Use enum instead, something like this:

userState with values ["Created", "Updated"] etc.

Reply:
@visingla-ms 
As discussed offline, we're just creating a stable version for the preview API versions which are already available and no new changes are being pushed here.
PFB the previous PRs:
https://github.com/Azure/azure-rest-api-specs/pull/22638
https://github.com/Azure/azure-rest-api-specs/pull/23343
https://github.com/Azure/azure-rest-api-specs/pull/23602
https://github.com/Azure/azure-rest-api-specs/pull/23650
https://github.com/Azure/azure-rest-api-specs/pull/24031